astro/src/content/quickstarts/quickstart-javascript-nuxt-web.mdx

```diff
@@ -0,0 +1,194 @@
+
+Create a new file named `/server/api/auth/[...].ts` and copy the following code for the ChangeBank application.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/server/api/auth/[...].ts"}
+  lang="ts" />
+
+## App Customization
+
+### Styles
+
+Create a new file named `/complete-application/assets/css/global.css` and copy the below CSS for the ChangeBank application.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/assets/css/global.css"}
+  lang="css" />
+
+The above styles will be imported into a new layout that you will create called `/complete-application/layouts/default.vue`. Update this file to match the code below. This will allow the rest of the pages within our application and the layout itself to use the global styles. This is the first time that you will use the [useAuth](https://sidebase.io/nuxt-auth/v0.6/application-side/session-access-and-management) Composable from Sidebase. The status can then be used to determine what links should be shown in the layout.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/layouts/default.vue"}
+  lang="ts" />
+
+### Login Button
+
+Create a new file in `/complete-application/components/LoginButton.vue` that will be used for a button component.
+
+For this button you can use the `signIn` and `signOut` functions from the `useAuth` composable. Copy the below code for the ChangeBank application into `/complete-application/components/LoginButton.vue`.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/components/LoginButton.vue"}
+  lang="ts" />
+
+### Login Link
+
+Create a new file in `/complete-application/components/LoginLink.vue` that will be used for a link component. For this link you can use the the same `signIn` function from the `useAuth` composable as you did in the button above.
+
+Copy the below code for the ChangeBank application into `/complete-application/components/LoginLink.vue`.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/components/LoginLink.vue"}
+  lang="ts" />
+
+### Home Page
+
+Update the file `/complete-application/pages/index.vue` which will have the Homepage details. Not much here just an image and a Login Link.
+
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/pages/index.vue"}
+  lang="ts" />
+
+
+### Account Page
+
+Create a new file `/complete-application/pages/account.vue` which will have some fake Account details.
+
+One special note here is that if you leave the auth `globalAppMiddleware` enabled in `nuxt.config.ts` you will need to authenticate to access this page.
+
+Here's the contents of `/complete-application/pages/account.vue`.
+<RemoteCode url={frontmatter.codeRoot + "/complete-application/pages/account.vue"}
+  lang="ts" />
+
+### Make Change Page
+
+Finally, we'll add some business logic for logged in users to make change with the following code in `/complete-application/pages/account.vue`:
```

Comment:
Am I just supposed to add this to the bottom of the account .vue page? I think it should be makechange.vue